zarpaint/napari.yaml

```diff
@@ -0,0 +1,39 @@
+name: zarpaint
+schema_version: 0.1.0
+contributions:
+  commands:
+  - id: zarpaint.add_points_3d_with_alt_click
+    title: Create add_points_3d_with_alt_click
+    python_name: zarpaint:add_points_3d_with_alt_click
+  - id: zarpaint.set_axis_labels
+    title: Create set_axis_labels
+    python_name: zarpaint:set_axis_labels
+  - id: zarpaint.DimsSorter
+    title: Create Dims Sorter
+    python_name: zarpaint:DimsSorter
+  - id: zarpaint.create_labels
+    title: Create create_labels
+    python_name: zarpaint:create_labels
+  - id: zarpaint.watershed_split
+    title: Create watershed_split
+    python_name: zarpaint:watershed_split
+  - id: zarpaint.get_reader
+    title: Get Reader
+    python_name: zarpaint.reader:zarr_tensorstore
```

Comment:
```suggestion
```

Do both readers and widgets need actions?

Reply:
Yep, at this stage you need both a `command` contribution and an associated `reader`/`widget` contribution.